### ISSUE TITLE
What 's the mlm accuracy of pretrained model? 

### ISSUE THREAD

**Issue by l126t** [huggingface/pytorch-pretrained-BERT#173]
What 's the mlm accuracy of pretrained model? In my case, I find  the scores of candidate in top 10 are very close，but most are not suitable. Is this the same prediction as Google's original project?

_Originally posted by @l126t in https://github.com/huggingface/pytorch-pretrained-BERT/issues/155#issuecomment-452195676_

**Comment by thomwolf** [comment]
Hi, we didn't evaluate this metric. If you do feel free to share the results.
Regarding the comparison between the Google and PyTorch implementations, please refere to the included Notebooks and the associated section of the readme.
